bootstrap/004_labels.sql

```diff
@@ -11,6 +11,9 @@ BEGIN
         UPDATE INTERNAL.LABELS SET LABEL_MODIFIED_AT = CURRENT_TIMESTAMP WHERE LABEL_MODIFIED_AT IS NULL;
     END IF;
 
+    -- Recreate the view to avoid number of column mis-match. Should be cheap and only run on install/upgrade, so it's OK
+    -- if we run this unnecessarily.
+    CREATE VIEW CATALOG.LABELS IF NOT EXISTS AS SELECT * FROM INTERNAL.LABELS;
```

Comment:
needs to be `create or replace` and not have `if not exists`, else this won't recreate the view, right?

Reply:
fixed